Library/Models/MetricCounts.cs

```diff
@@ -0,0 +1,30 @@
+﻿//    This file is part of Nfield.SDK.
+//
+//    Nfield.SDK is free software: you can redistribute it and/or modify
+//    it under the terms of the GNU Lesser General Public License as published by
+//    the Free Software Foundation, either version 3 of the License, or
+//    (at your option) any later version.
+//
+//    Nfield.SDK is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//    GNU Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with Nfield.SDK.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace Nfield.Models
+{
+    public class MetricCounts
+    {
+        /// <summary>
+        /// The number of times an interview reached the warn threshold
+        /// </summary>
+        public int Warn { get; set; }
+
+        /// <summary>
+        /// The number of times an interview reached the kill threshold
```

Comment:
IIRC the downloadable logs call these "Blocked", it might be better to keep consistency (also for the public API)

Reply:
I changed the public API: https://github.com/NIPOSoftwareBV/nfield-source/pull/6575